content/en/docs/updates/2022_08_26_update_unit_tests_with_test_utils.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Update project unit tests to use @axway/api-builder-test-utils
+date: 2022-09-16
+---
+
+## Why are we making this change
+
+In the [Villasimius](/docs/release_notes/villasimius/) release, we introduced a new `Project` utility in [@axway/api-builder-test-utils](https://www.npmjs.com/package/@axway/api-builder-test-utils) to reduce the amount of code that users have to manage in their project unit tests. When this feature is used, it allows us to provide unit test optimisations for new {{% variables/apibuilder_prod_name %}} Core features through regular updates without users having to make manual changes (such as this one) to receive them.
+
+## How does this impact my service
+
+To benefit from this feature, existing project's unit tests must be modified as described below. Once the modifications have been made, future updates to [@axway/api-builder-test-utils](https://www.npmjs.com/package/@axway/api-builder-test-utils) will be able to be seen from the **Updates** tab.
+
+New projects have this feature enabled by default.
+
+## Upgrade existing services
+
+This feature requires the [Villasimius](/docs/release_notes/villasimius/) release of {{% variables/apibuilder_prod_name %}} Core, so ensure you have installed all updates before continuing. This guide also assumes you have followed the previous update outlined in [Replace the request dev-dependency in project unit tests](/docs/updates/2021_12_17_update_to_remove_request_module)
+
+### Install @axway/api-builder-test-utils
+
+Install [@axway/api-builder-test-utils](https://www.npmjs.com/package/@axway/api-builder-test-utils) (at least 1.6.0) as a dev-dependency to your project.
+
+```bash
+npm install @axway/api-builder-test-utils@1.6.0 --save-dev
+```
+
+### Import @axway/api-builder-test-utils
+
+In all of your project's unit tests, find this code:
+
+```javascript
+const { startApiBuilder, stopApiBuilder } = require('./_base');
+```
+
+And replace it with:
+
+```javascript
+const { Runtime } = require('@axway/api-builder-test-utils');
+```
+
+### Remove _base.js
+
+The file `/test/_base.js` is now no longer used, and can be deleted from your project.
+
+### Replace startApiBuilder with Runtime
+
+In all of your project's unit tests, find the `before` block of code:
+
+```javascript
+apibuilder = await startApiBuilder();
+const apikey = apibuilder.config.apikey;
+```
+
+Repalce `startApiBuilder` with `Runtime`.
+```javascript
+runtime = new Runtime();
+```
+Remove `apikey` and `port` reference from code, its now handled in `Runtime`.
+
+### Remove stopApiBuilder
+
+Delete the following code:
+
+```javascript
+/**
+ * Stop API Builder after the tests.
+ */
+after(() => stopApiBuilder(apibuilder));
+```
+
+### Remove got client
+`got` client is replaced with `runtime.request`, which supports `url`, `method`, `headers`, and `body` as options.
+
+```javascript
+runtime.request({
```

Comment:
**issue:** there's no before/after examples. The example here is not correct since it is passing body and content-type as headers to a request for apibuilderPing.json